packages/client/hmi-client/src/components/model/petrinet/tera-initial-entry.vue

```diff
@@ -1,34 +1,40 @@
 <template>
 	<div>
 		<header>
-			<div>
+			<div class="flex">
 				<strong>{{ initialId }}</strong>
 				<span v-if="name" class="ml-1">{{ '| ' + name }}</span>
 				<template v-if="unit">
 					<label class="ml-2">Unit</label>
 					<span class="ml-1">{{ unit }}</span>
 				</template>
+				<template v-if="concept">
+					<label class="ml-auto">Concept</label>
+					<span class="ml-1">{{ concept }}</span>
+				</template>
 			</div>
-			<span v-if="description" class="ml-4">{{ description }}</span>
+			<span v-if="description" class="description">{{ description }}</span>
 		</header>
-		<main>
-			<span class="expression">
+		<template v-if="isEmpty(modelConfiguration.inferredParameterList)">
```

Comment:
FYI we will need to revisit this, strictly speaking you still have initial values, they just shouldn't be editable.

Reply:
I am just hiding the expression and some buttons, should I just show a read only expression?

Reply:
I want to clear up with TA3 what exactly do they use when we send them a calibrated model-config.

So for example in SIR model the dill file has some functor representation of the parameters beta, gamma, S0, I0, R0.
Say our S has expression `S0 - 10` ... it is unclear to me whether it makes sense or not to change it to say `1000` as that basically would void the calibration, but technically there's nothing wrong with doing that.

Moreover, currently for the calibrated config `initialSemanticList` is empty so also not entirely sure what we want to do there.